xalia/Win32/Win32Connection.cs

```diff
@@ -306,6 +306,11 @@ internal UiDomElement CreateElement(ElementIdentifier id)
                 result.AddProvider(new AccessibleProvider(root_hwnd, result, id.acc, id.child_id));
             }
 
+            if (!(id.prov is null) && (id.runtime_id is null))
+            {
+                result.AddProvider(new UiaProvider(root_hwnd, result, id.prov));
+            }

```

Comment:
This confused me a little bit, but just so I understand, the sequence is:
- If there's a UIA provider with a runtime ID, add that.
- If there's an IAccessible or an IAccessible2, add that.
- If there was a UIA provider but it had no runtime ID, add it now.

Reply:
Correct. Mostly the reason for that is, I want to keep using MSAA for enumeration with Chrome. Although it shouldn't make much difference.